src/grafana_dashboards/postgresql-metrics.json

```diff
@@ -216,7 +216,8 @@
           "format": "time_series",
           "intervalFactor": 2,
           "legendFormat": "",
-          "refId": "A"
+          "refId": "A",
+          "instant": true
```

Comment:
all of the `"instant": true` additions are for the `singlestat` type panels: sometimes, if the user re-deployed the pg charm while monitoring, the value series get split in two (before and after). For graph panels it's mostly self-explanatory, but for single stats, the panel gets bloated with chars (it tries to print both values with the job names on it, see before screeshot of `Start Time` for an example). This way, we only fetch the instant current value.